tests/integration/llm/client.py

```diff
@@ -205,6 +219,18 @@ def test_sd_handler(model, model_spec):
             for memory in memory_usage:
                 assert float(memory) / 1024.0 < spec["max_memory_per_gpu"]
 
+def test_ft_raw_handler(model, model_spec):
+    if model not in model_spec:
+        raise ValueError(
+            f"{model} is not one of the supporting models {list(ft_raw_model_spec.keys())}"
+        )
+    spec = model_spec[model]
+    for batch_size in spec['batch_size']:
+        print(f"testing ft_handler with model: {model}, batch_size: {batch_size} ")
+        req = {"inputs" : t5_batch_generation(batch_size)}
+        res = send_json(req)
+        res = res.json()
+        assert len(res) == batch_size
```

Comment:
Could we also print the result and gpu memory usages? For different TP degree, we should also limit the memory per gpu to avoid some (load on single GPU) cases.

Reply:
I will gather the data for this and add.

Reply:
Added. logs here - https://github.com/deepjavalibrary/djl-serving/actions/runs/4140117446/jobs/7158467593